client/blocks/reader-featured-video/index.jsx

```diff
@@ -55,6 +62,12 @@ class ReaderFeaturedVideo extends React.Component {
 
 	handleThumbnailClick = ( e ) => {
 		e.preventDefault();
+		this.props.onThumbnailClick();
```

Comment:
I think @blowery means that clicking on any area of the card should bring you to fullpost and so if thumbnail-specific behavior is turned off then fullpost clickhandler will get invoked anyway.

Feel free to keep it, its a minor detail